lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -605,6 +662,27 @@ def add_managed_billing(xml, options)
         end
       end
 
+      # Validates Electronic Check Serial number based on bin value provided
+      def validate_ecp_check_serial(serial_number, bin)
+        (bin.eql?('000001') && serial_number.length.eql?(9)) || (bin.eql?('000002') && serial_number.length.eql?(6))
```

Comment:
I'm actually a little confused here. Are we only accepting payment methods with a bin that is one of these 2 values? @jessiagee?

Reply:
@naashton From Erik: "Really only 000001.  I know we have the logic in the AM code for looking at those BIN values based on MID length but from a Spreedly perspective we only support the Stratus platform which is 000001."

Reply:
Also from the spec doc: "The Orbital Gateway supports Electronic Check Processing (ECP) for eligible merchants. This method of payment is only available to Stratus platform merchants (BIN 000001)." So I suppose we could actually just check if BIN == 000001 and if not than we can't do ECP.

Reply:
Since the code on line 675 will be removed as per previous comment hence this function is not required anymore so it will be removed.